documentation/feature/missileguidance.md

```diff
@@ -10,6 +10,19 @@ parent: wiki
 
 Adds the AMG framework, for more information about it refer to the [AMG framework documentation] (http://ace3mod.com/wiki/framework/advanced-missile-guidance.html)
 
-## 2. Dependencies
+## 2. Usage
+
+### 2.1 Switching fire mode
+- Press <kbd>CTRL</kbd> + <kbd>TAB</kbd> the LEDs on the right of the UI will change.
```

Comment:
Are those UI elements LED's? If there's no better word for it would be `LED's` and not `LEDs`.